packages/i18n/locales/cs

```diff
@@ -0,0 +1,457 @@
+      "easyToUse": "Snadno použitelné rozhraní",
+      "easyToUseExplanation": "Rozhraní DAO DAO zpřístupňuje DAOs všem....",
+      "joinTheCommunity": "Připojte se ke komunitě.",
+      "longTagline": "Jednoduchá, promyšlená a bezplatná správa organizace. Vytvořeno s láskou společností DAO DAO, od společnosti Juno.",
+      "poweredByJuno": "Powered by Juno",
+      "proposalsCreated": "Vytvořené návrhy ",
+      "proposeAndVote": "Návrhy a hlasy",
+      "readMore": "Zobrazit více",
+      "shortTagline": "Organizace pro všechny.",
+      "transparencyExplanation": "Každý se může podívat, jaká rozhodnutí navrhujete a kdo hlasoval pro nebo proti.",
+      "transparentGovernanceOnChain": "Transparentní správa v blockchainu",
+      "usdcTotalValue": "Hodnota USDC",
+      "whatIsADao": "Co je DAO?",
+      "whatIsADaoExplanation": "DAO je decentralizovaná autonomní organizace demokraticky řízená svými členy....."
+    },
+    "success": {
+      "addedToken": "Přidání tokenů doKeplru.",
+      "daoCreatedPleaseWait": "Organizace byla vytvořena. Na stránku budete přesměrováni, až bude připravena.....",
+      "proposalClosed": "Úspěšně uzavřeno.",
+      "proposalExecuted": "Úspěšně provedeno",
+      "voteCast": "Hlasování proběhlo úspěšně."
+    },
+    "title": {
+      "404": "404 - Stránka nebyla nalezena",
+      "500": "500 - Interní problém serveru",
+      "actions_one": "Akce",
+      "actions_other": "Akce",
+      "addTreasuryToken": "Přidání tokenu pokladny",
+      "addresses": "Adresy",
+      "allVotes": "Všechny hlasy",
+      "apr": "RPSN",
+      "aprReward": "RPSN odměny",
+      "balanceUnstaked": "Zůstatek (nepřidělený {{name}})",
+      "beforeYouEnter": "Před vstupem",
+      "casting": "Vyčkejte",
+      "chooseAStructure": "Zvolte strukturu",
+      "chooseTokenAmount": "Zvolte množství tokenů",
+      "claimTokens": "Vyberte tokeny",
+      "configureVoting": "Konfigurace hlasování",
+      "configureWalletToContinue": "Konfigurace peněženky pro pokračování",
+      "contractVersion": "Verze smlouvy",
+      "createADAO": "Vytvořte DAO",
+      "createAProposal": "Vytvořte návrh",
+      "currentlyUnstaking": "Aktuálně odemykáte",
+      "custom": "Vlastní",
+      "dao": "DAO",
+      "daoTreasury": "DAO pokladna",
+      "daos": "DAOs",
+      "describeTheDAO": "Popis organizace",
+      "details": "detajly",
+      "documentation": "Dokumentace",
+      "executeSmartContract": "Prosadit smlouvu",
+      "feedback": "Zpětná vazba",
+      "governanceToken": "Token organizace",
+      "history": "Historie",
+      "home": "Domů",
+      "info": "Info",
+      "instantiateSmartContract": "Vytvoření instance kontaktu",
+      "manageMembers": "Spravujte členy",
+      "memberVotingPowers": "Hlasovací právo členů",
+      "members": "Členi",
+      "migrateSmartContract": "Migrace smlouvy",
+      "mint": "Vytvořit",
+      "needWalletToContinue": "Abyste mohli pokračovat, musíte připojit peněženku",
```

Comment:
Abyste
https://proofreading.cz/cestina-pod-lupou/abyste-nikoli-aby-jste-ani-abyjste/

Reply:
No však to tam je Omegalul